src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php

```diff
@@ -58,22 +67,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
             throw new \InvalidArgumentException('The provided connection is not an instance of the Doctrine DBAL connection.');
         }
 
-        if (true !== $input->getOption('dump-sql')) {
-            $output->write('ATTENTION: This operation should not be executed in a production environment.' . PHP_EOL . PHP_EOL);
+        if (true !== $input->getOption('dump-sql') && !$input->getOption('force')) {
+            $output->write('ATTENTION: This operation should not be executed in a production environment. Please use "--force" to execute the command.' . PHP_EOL . PHP_EOL);
+            return self::RETURN_CODE_NO_FORCE;
         }
 
         $schema = new RepositorySchema;
+
         try {
             if ($input->getOption('drop')) {
                 foreach ($schema->toDropSql($connection->getDatabasePlatform()) as $sql) {
                     if (true === $input->getOption('dump-sql')) {
                         $output->writeln($sql);
                     } else {
-                        try {
-                            $connection->exec($sql);
-                        } catch (\Exception $e) {
-                            $output->writeln($e->getMessage());
-                        }
```

Comment:
hmm not sure ... I guess we wanted to give more context in the error output than we get from just outputting the error message.